### ISSUE TITLE
As a user, I would like to see permit purchase date year on the permit instead of season start date year.

### ISSUE THREAD

**Issue by lpendem** [nciinc/fs-permit-platform#277]

**Comment by sadlerw** [comment]
@ASprinkle we are thinking maybe we need to talk about this one a bit more and possible get LEO feedback on which date should be on the permit.

**Comment by sadlerw** [comment]
this ok?
![Screen Shot 2018-01-24 at 2.20.52 PM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/155cb198-95eb-481b-ba58-ec00f81385e0)

**Comment by sadlerw** [comment]
this better:
[https://app.zenhub.com/files/106473503/44ff6813-bfc1-4e61-9e63-1df8d0eb5a7d/download](https://app.zenhub.com/files/106473503/44ff6813-bfc1-4e61-9e63-1df8d0eb5a7d/download)

**Comment by bryanmcfadden** [comment]
@sadlerw - That's much better! 
I think adding the space in the issue date and breaking the "N" and "A" into 2 separate pieces made a world of difference.

**Comment by ASprinkle** [comment]
Yes! can we have a comma between the day and the year? just like in expiration date box.

**Comment by ASprinkle** [comment]
Why are we working on the Max Tree Height when this story addresses issue date year?  Have I missed something? @sadlerw your question about LEO input - have you received an answer? @MelissaBraxton do you have info from your research?

**Comment by sadlerw** [comment]
It isn't the max tree height @ASprinkle - we noticed that the N A in the new boxes wasn't formatting correctly in firefox so added a task to update that since we are updating the issue year.

**Comment by ASprinkle** [comment]
Ok, thanks for clarifying

**Comment by sadlerw** [comment]
Also comma is added - check out the pdf I linked above.  thanks!

**Comment by ASprinkle** [comment]
Great!! Looks good!
